networks.json

```diff
@@ -325,7 +325,7 @@
             },
             {
               "indexed": true,
-              "name": "orderIndexes",
+              "name": "orderIds",
```

Comment:
If we revert to using Index, this should be `orderIndices` not `Indexes` since indices is plural of index.

Reply:
Both is considered good english:
https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12

Reply:
Although this reference is about a _stock index_ (cf. nasdaq) where we are referring to a mathematical index. I believe the last sentence in the article you reference here states this as well.

```
Both "indexes" and "indices" are acceptable plural forms of the word "index" or to refer to more than one index. Index is one of those rare words that have two different plurals in English. "Indices" is originally a Latin plural, while "Indexes" has taken the English way of making plurals, using –s or –es. Though both are still widely used, they take on different usage in their senses. "Indices" is used when referring to mathematical, scientific and statistical contexts.
```

Reply:
You are the native one :) 
I might be wrong, but I've always seen indexes when using it for IT

Reply:
:+1: for indices, it sounds more natural to me as well.

Reply:
@anxo - you have been reading poorly written IT documentation.